### PR TITLE
Html and css

### PR DESCRIPTION
**Pull request by Bensonre**
I haven't added stuff for errors or the background image but will be simple fix later. Game works with current code and set up. Chase since you will be working on the same files I suggest you do a a deep review to get to know the code.  the only stuff added to the JS was to make the play board copy during ship placement mimic regular player board. Small fix that does need to be done.

td{
background-color:lightgray;
} 
needs to be added to CSS because the the player board in attack mode shows up as white. this makes it so you can't see the spaces the AI attacks and misses.This should fix but I will leave it to you chase.

### CODE REVIEW

**Comment by Bensonre** [comment]
the errors ^ seem to have started happening when chase added image files. I don't think it was this code.
see ( https://travis-ci.com/cs361fall2018/project-dixonallstars/builds ) for details.

**Comment by xMestas** [comment]
You are sure that this issue is from the image files?  Should I ask about it on Slack?

File: src/main/java/assets/game.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ function markHits(board, elementId, surrenderText) {
         else if (attack.result === "HIT")
             className = "hit";
         else if (attack.result === "SUNK")
-            className = "hit"
+            className = "sunk"
```
Comment (xMestas) [inline src/main/java/assets/game.js]: Ahhh this is why they weren't showing up as orange in the previous sprint

File: src/main/java/assets/game.js [hunk 2]
```diff
@@ -34,14 +34,17 @@ function markHits(board, elementId, surrenderText) {
 function redrawGrid() {
     Array.from(document.getElementById("opponent").childNodes).forEach((row) => row.remove());
     Array.from(document.getElementById("player").childNodes).forEach((row) => row.remove());
+    Array.from(document.getElementById("player_copy").childNodes).forEach((row) => row.remove());
```
Comment (xMestas) [inline src/main/java/assets/game.js]: Is this because there is a top and lower version of the player grid?

File: src/main/java/assets/game.js [hunk 3]
```diff
@@ -123,10 +122,22 @@ function place(size) {
             }
             cell.classList.toggle("placed");
         }
+  }
+function place(size) {
```
Comment (xMestas) [inline src/main/java/assets/game.js]: Add some comments to this file.  I am having a really hard time following these placement functions.  Maybe this is where the hover thing is coming from?

File: src/main/java/assets/game.js (hunk 2 above)
Comment (Bensonre) [reply src/main/java/assets/game.js]: yeah, exactly

File: src/main/java/assets/game.js (hunk 3 above)
Comment (Bensonre) [reply src/main/java/assets/game.js]: I just moved the placement function he already had in there. I was having the issue before i did this but I will add comments so It makes sense.

**Comment by Bensonre** [comment]
yeah Im pretty sure thats where the issues are coming from if you look at the link i had there thats when the builds stop working. Asking in slack couldn't hurt though

File: src/main/java/assets/game.js (hunk 3 above)
Comment (chase-stubbs) [reply src/main/java/assets/game.js]: ++

**Comment by xMestas** [comment]
Implements #16

**Pull request merged**